Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -149,7 +148,10 @@ public void Install(IWindsorContainer container, IConfigurationStore store)
 
             RegisterWindowsHooks(container);
 
-            RegisterHotkeyFactory(container);
+            container.Register(Component.For<HotkeyFactory>()
+                .LifestyleSingleton());
+            container.Register(Component.For<ITestEngine>().ImplementedBy<TestEngine>()
+                .LifestyleSingleton());
```

Comment:
this registration should already be covered by the interface convention ...

Reply:
I believe the default lifestyle is `Transient`, is it not?

Reply:
It is

Reply:
Without this change `ITestEngine` was registered as `Transient`, which caused the #3590 bug.